### ISSUE TITLE
Drop cachier requirement from build tools

### ISSUE THREAD

**Issue by jamesaoverton** [OBOFoundry/OBOFoundry.github.io#1379]
@cmungall can we drop the requirement for `cachier` from our build tools? It's not really necessary. I've had consistent problems installing it on macOS, and now it seems to be breaking Travis:

https://travis-ci.com/github/OBOFoundry/OBOFoundry.github.io/builds/207782121

**Comment by cmungall** [comment]
yes, drop it. I didn't even realize we were using it here

We used to have a dependency on it in ontobio, but removed it for the same reasons.

**Comment by jamesaoverton** [comment]
Thanks. I fixed this in 00913b5d459efa1277b00a10a4d52a8d25a61cf7 directly on `master`.

(Just to be clear: I meant to make a PR but forgot to branch. We had branch protection for `master` turned on before, which would have prevented my push to `master`. But I turned protection off for administrators so that I could merge some PRs while Travis was broken, and I also forgot about that. Now I've turned branch protection back on.)
